cleanlab/internal/multilabel_utils.py

```diff
@@ -64,3 +64,34 @@ def get_onehot_num_classes(labels, pred_probs=None):
             "wrong format for labels, should be a list of list[indices], please check the documentation in find_label_issues for further information"
         )
     return y_one, num_classes
+
+
+def int2onehot(labels, K) -> np.ndarray:
+    """Convert list of lists to a onehot matrix for multi-labels
+    Parameters
+    ----------
+    labels: list of lists of integers
+      e.g. [[0,1], [3], [1,2,3], [1], [2]]
+      All integers from 0,1,...,K-1 must be represented.
+    K: int
+      The number of classes."""
+
+    from sklearn.preprocessing import MultiLabelBinarizer
+
+    mlb = MultiLabelBinarizer(classes=range(K))
+    return mlb.fit_transform(labels)
+
+
+def onehot2int(onehot_matrix) -> list:
+    """Convert a onehot matrix for multi-labels to a list of lists of ints
+    Parameters
+    ----------
+    onehot_matrix: 2D np.ndarray of 0s and 1s
+      A one hot encoded matrix representation of multi-labels.
+    Returns
```

Comment:
```suggestion
    onehot_matrix: 2D np.ndarray of 0s and 1s
      A matrix representation of multi-label classification labels in a binarized format as a multi-hot vector for each example. 
      The entries in this vector are 1 for each class that applies to this example and 0 otherwise.
    
    Returns
```